index.html

```diff
@@ -291,6 +291,42 @@ <h2>Skills</h2>
             </ul>
         </div>
     </section>
+    <section class="contact-me">
+        <div class="contact-me-text1">
+            <h2>Contact me</h2>
+        </div>
+        <div class="contact-me-text2">
+            <p>If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding.
+                I’d love to help with it </p>
+        </div>
+        <div class="form-section">
+            <form action="https://formspree.io/f/id1" method="post" id="id1">
+                <ul class="form-list">
+                    <li>
```

Comment:
- [ ] check [ Formspree service.](https://formspree.io/html) to make sure you are not making any mistake. I got the error on the image below.

![form](https://user-images.githubusercontent.com/53124548/134156687-65de4dde-80e7-48e6-9260-5d1ca6e9f093.png)